src/recovery/cubic.rs

```diff
@@ -175,6 +179,7 @@ fn on_packet_acked(
     } else {
         // Congestion avoidance.
         let ca_start_time;
+        let cubic = &mut r.cubic_state;
```

Comment:
It doesn't seem like this variable provides much benefit and is not used consistently, so IMO we can just remove this and use `r.cubic_state` directly.

Reply:
Done